src/components/Sidebar/Sidebar.tsx

```diff
@@ -63,6 +66,7 @@ const tabBarCss = css`
     border-bottom: none;
     width: calc(100% / 3);
     text-align: center;
+    overflow-x: auto;
```

Comment:
We probably want the parent to strictly enforce its borders instead of using auto overflow.